### PR TITLE
T16344

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T16344

### CODE REVIEW

**Comment by tchx84** [comment]
@ptomato @xjuan do this makes sense?

File: js/app/downloader.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ const Soup = imports.gi.Soup;
 const Config = imports.app.config;
 const Utils = imports.app.utils;
 
+const CHUNK_SIZE = 1024 * 128;
```
Comment (ebassi) [inline js/app/downloader.js]: I don't want to bikeshed on the number, but I'd probably either downsize this to 64 or up it to 256 kB, with a preference for the latter.

**Comment by ebassi** [comment]
Nice catch!

**Comment by ebassi** [comment]
`Utils.CHUNK_SIZE` existed until https://phabricator.endlessm.com/T15257 was fixed.

**Comment by ebassi** [comment]
CR pass.

**Pull request merged**